controllers/sessions.js

```diff
@@ -0,0 +1,90 @@
+const fs = require('fs')
+const Sessions = require('../data/sessions.json')
+
+
+
+const listSession = (req, res) => {
+
+    res.json(Sessions);
+
+}
+
+
+const createSession = (req, res) => {
+
+    const newSession = {
+        idSession: req.query.idSession,
+        idPsicologist: req.query.idPsicologist,
+        idPostulant: req.query.idPostulant,
+        date: req.query.date,
+        state: req.query.state,
+    }
+
+    Sessions.push(newSession)
+
+    fs.writeFile('./data/sessions.json', JSON.stringify(Sessions), {}, (error) => {
+        if (error) {
+            res.status(400).send(error)
+        } else {
+            res.status(201).json(newSession)
+        }
+    })
+}
+
+
+const updateSession = (req, res) => {
+    let updatedSession
+    const updatedSessions = Sessions.map((session) => {
+        if (session.idSession == req.query.idSession) {
```

Comment:
What happens if the id does not exist on the list?  
The list should not be updated and the endpoint should return with a 404 error.